### ISSUE TITLE
Badge count

### ISSUE THREAD

**Issue by alexcroox** [capacitor-community/proposals#101]
## Plugin Request

**Name**: Capacitor Badge
**Package**: `@capacitor-community/capacitor-badge`

### Platform(s)

ios/android

### Existing Solutions

https://github.com/katzer/cordova-plugin-badge

### Description

Ability to set the unread badge on the homescreen icon without relying on push notifications to set it. 

This is useful when you have a number of unread messages from an API call that you want to programmatically set the unread badge count. 

It would be great if this could also work with background tasks so it can be updated from periodic API calls when the app isn't in focus.

**Comment by robingenz** [comment]
Give it a try: https://github.com/robingenz/capacitor-badge
I appreciate any feedback.

**Comment by alexcroox** [comment]
Oh wow thank you so much for this!! I will give it a spin first thing Monday!

**Comment by robingenz** [comment]
I'm glad to hear that! Please be aware that I have already created the plugin for Capacitor 3, so Capacitor 2 is not supported.
If you haven't updated your app to Capacitor 3 yet, I can only recommend it. 😄 
The RC is very stable.

**Comment by alexcroox** [comment]
Not a problem I’m working on a new app and have already migrated old code from Cap 2 to 3 👍

**Comment by alexcroox** [comment]
@robingenz seems to be working great thanks!

**Comment by riderx** [comment]
@robingenz i added your plugin to [awesome-capacitor](https://github.com/riderx/awesome-capacitor) if that is useful please take time to star the repo as well :)

**Comment by robingenz** [comment]
@riderx Of course, I can only speak for myself here: I would appreciate it if you avoid this spam in the future. Everyone who watches this repo has received more than 40 notifications. I am not sure if that was necessary.

**Comment by riderx** [comment]
@robingenz yes i'm sper sorry !
Your are not the only one to tel me that, I totally understand.
i never watch a full repo, I didn't think about this ...
I was just happy to share, it was a huge work.

**Comment by d8nytelweb** [comment]
ShortcutBadger installed the plugin and initialized the value but for some reason it never shows me the badges, help
